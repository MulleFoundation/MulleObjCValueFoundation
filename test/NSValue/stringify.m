#ifndef __MULLE_OBJC__
# import <Foundation/Foundation.h>
#else
# import <MulleObjCValueFoundation/MulleObjCValueFoundation.h>
#endif


int   main( void)
{
   NSValue   *value;
   struct abcd { char a; int b; char c; double d; } x = { '1', 2, 3, 4 };

   value = [NSValue valueWithBytes:&x
                          objCType:@encode( struct abcd)];
   mulle_printf( "%@\n", value);

   return( 0);
}
