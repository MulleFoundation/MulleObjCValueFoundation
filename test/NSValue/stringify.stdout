{ '1', 2, '\003', 4 }
